apps/emqx_auth_mnesia/test/emqx_auth_mnesia_SUITE.erl

```diff
@@ -295,8 +296,27 @@ t_clientid_rest_api(_Config) ->
                   }, get_http_data(Result3)),
 
     {ok, Result4} = request_http_rest_list(["auth_clientid"]),
-
-    ?assertEqual(3, length(get_http_data(Result4))),
+    #{<<"data">> := Data4, <<"meta">> := #{<<"count">> := Count4}}
+        = emqx_json:decode(Result4, [return_maps]),
+
+    ?assertEqual(3, Count4),
+    ?assertEqual(3, length(Data4)),
+
+    UserNameParams = [ #{<<"username">> => ?USERNAME, <<"password">> => ?PASSWORD}
```

Comment:
Tiny nit: maybe define a different number of usernames to make it clear that it's indeed not selecting clientids, which also sum to 3 above?